src/Symfony/Component/Mime/Part/DataPart.php

```diff
@@ -28,7 +28,6 @@ class DataPart extends TextPart
     private $filename;
     private $mediaType;
     private $cid;
-    private $handle;
 
     /**
      * @param resource|string $body
```

Comment:
Should not we remove this phpdoc in order to inherit doc of `TextPart` ? `@param resource|string $body Use file:// to defer loading the file until rendering`